types/core/config.d.ts

```diff
@@ -1,141 +1,1045 @@
+     */
+    chat: ConstructorOf<ChatLog>;
+
+    /**
+     * @defaultValue `CombatTracker`
+     */
+    combat: ConstructorOf<CombatTracker>;
+
+    /**
+     * @defaultValue `CompendiumDirectory`
+     */
+    compendium: ConstructorOf<CompendiumDirectory>;
+
+    /**
+     * @defaultValue `SceneControls`
+     */
+    controls: ConstructorOf<SceneControls>;
+
+    /**
+     * @defaultValue `Hotbar`
+     */
+    hotbar: ConstructorOf<Hotbar>;
+
+    /**
+     * @defaultValue `ItemDirectory`
+     */
+    items: ConstructorOf<ItemDirectory>;
+
+    /**
+     * @defaultValue `JournalDirectory`
+     */
+    journal: ConstructorOf<JournalDirectory>;
+
+    /**
+     * @defaultValue `MacroDirectory`
+     */
+    macros: ConstructorOf<MacroDirectory>;
+
+    /**
+     * @defaultValue `MainMenu`
+     */
+    menu: ConstructorOf<MainMenu>;
+
+    /**
+     * @defaultValue `SceneNavigation`
+     */
+    nav: ConstructorOf<SceneNavigation>;
+
+    /**
+     * @defaultValue `Notifications`
+     */
+    notifications: ConstructorOf<Notifications>;
+
+    /**
+     * @defaultValue `Pause`
+     */
+    pause: ConstructorOf<Pause>;
+
+    /**
+     * @defaultValue `PlayerList`
+     */
+    players: ConstructorOf<PlayerList>;
+
+    /**
+     * @defaultValue `PlaylistDirectory`
+     */
+    playlists: ConstructorOf<PlaylistDirectory>;
+
+    /**
+     * @defaultValue `SceneDirectory`
+     */
+    scenes: ConstructorOf<SceneDirectory>;
+
+    /**
+     * @defaultValue `Settings`
+     */
+    settings: ConstructorOf<Settings>;
+
+    /**
+     * @defaultValue `Sidebar`
+     */
+    sidebar: ConstructorOf<Sidebar>;
+
+    /**
+     * @defaultValue `RollTableDirectory`
+     */
+    tables: ConstructorOf<RollTableDirectory>;
+
+    /**
+     * @defaultValue `CameraViews`
+     */
+    webrtc: ConstructorOf<CameraViews>;
   };
 
   /**
-   * Available Weather Effects implemntations
+   * Available Weather Effects implementations
+   */
+  weatherEffects: {
+    leaves: any; // TODO: ConstructorOf<AutumnLeavesWeatherEffect>
+    rain: any; // TODO: ConstructorOf<RainWeatherEffect>
+    snow: any; // TODO: ConstructorOf<SnowWeatherEffect>
+  } & Partial<Record<string, any>>; // TODO: ConstructorOf<SpecialEffect>
+
+  /**
+   * Configuration for the WebRTC implementation class
    */
-  weatherEffects: any;
-};
+  WebRTC: {
+    /**
+     * @defaultValue `EasyRTCClient`
+     */
+    clientClass: any; // TODO: ConstructorOf<EasyRTCClient>
+
+    /**
+     * @defaultValue `50`
+     */
+    detectPeerVolumeInterval: number;
+
+    /**
+     * @defaultValue `20`
+     */
+    detectSelfVolumeInterval: number;
+
+    /**
+     * @defaultValue `25`
+     */
+    emitVolumeInterval: number;
+
+    /**
+     * @defaultValue `2`
+     */
+    speakingThresholdEvents: number;
+
+    /**
+     * @defaultValue `10`
+     */
+    speakingHistoryLength: number;
+  };
+} & Record<string, unknown>;
+
+declare namespace CONFIG {
+  // TODO: Where to put this?
```

Comment:
I would leave this exactly here, but rename it to `Config`, to be consistent with the `Const` namespace.

Reply:
Done.